Integrations/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM deephaven/java-and-python:local-build as install-wheels
+
+RUN set -eux; \
+    apt-get -qq update; \
+    apt-get -qq -y --no-install-recommends install liblzo2-2 curl; \
+    rm -rf /var/lib/apt/lists/*
+
+COPY deephaven-jpy-wheel/ /deephaven-jpy-wheel
+RUN set -eux; \
+    python3 -m pip install -q --no-cache-dir /deephaven-jpy-wheel/*.whl; \
+    rm -r /deephaven-jpy-wheel
+
+COPY deephaven-wheel/ /deephaven-wheel
+RUN set -eux; \
+    python3 -m pip install -q --no-cache-dir /deephaven-wheel/*.whl; \
```

Comment:
(cc @jmao-denver) I just noticed this wondering if you have any insights - if I'm reading this right, the wheel itself is 137kB, but the installed sized is 217MB?

```
3c7d9ad99d54   8 minutes ago   /bin/sh -c set -eux;     python3 -m pip inst…   217MB     
ec679d5275dd   8 minutes ago   /bin/sh -c #(nop) COPY dir:7dc2bc98d33cb7f8d…   137kB  
```
```
Step 5/10 : COPY deephaven-wheel/ /deephaven-wheel
 ---> ec679d5275dd
Step 6/10 : RUN set -eux;     python3 -m pip install -q --no-cache-dir /deephaven-wheel/*.whl;     rm -r /deephaven-wheel
 ---> Running in 046d3e02500b
+ python3 -m pip install -q --no-cache-dir /deephaven-wheel/deephaven-0.7.0-py2.py3-none-any.whl
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
+ rm -r /deephaven-wheel
Removing intermediate container 046d3e02500b
 ---> 3c7d9ad99d54
```

Reply:
Here are the sizes I get locally:
```shell
$ ls -la Integrations/build/context/*         
-rw-r--r--. 1 devin devin 761 Nov 12 12:03 Integrations/build/context/Dockerfile

Integrations/build/context/deephaven2-wheel:
total 16
drwxr-xr-x. 1 devin devin    66 Nov 12 12:03 .
drwxr-xr-x. 1 devin devin   120 Nov 12 12:03 ..
-rw-r--r--. 1 devin devin 12942 Nov 12 12:03 deephaven2-0.7.0-py3-none-any.whl

Integrations/build/context/deephaven-jpy-wheel:
total 352
drwxr-xr-x. 1 devin devin     94 Nov 12 12:03 .
drwxr-xr-x. 1 devin devin    120 Nov 12 12:03 ..
-rw-r--r--. 1 devin devin 356445 Nov 12 12:03 deephaven_jpy-0.7.0-cp37-cp37m-linux_x86_64.whl

Integrations/build/context/deephaven-wheel:
total 136
drwxr-xr-x. 1 devin devin     72 Nov 12 12:03 .
drwxr-xr-x. 1 devin devin    120 Nov 12 12:03 ..
-rw-r--r--. 1 devin devin 137119 Nov 12 12:03 deephaven-0.7.0-py2.py3-none-any.whl
```

Reply:
The large install size is due to dependency on numba and other stuff.